bitnami/airflow/templates/metrics/deployment.yaml

```diff
@@ -53,6 +53,9 @@ spec:
       {{- if .Values.metrics.priorityClassName }}
       priorityClassName: {{ .Values.metrics.priorityClassName }}
       {{- end }}
+      {{- if .Values.metrics.runtimeClassName }}
+      runtimeClassName: {{ .Values.metrics.runtimeClassName }}
+      {{- end }}
```

Comment:
It seems this one is not defined in the _values.yaml_

Reply:
Oh, priorityClassName wasn't defined for metrics too, fixed it